src/staticarrays_support.jl

```diff
@@ -0,0 +1,19 @@
+import StaticArrays: StaticArray, tuple_prod
+
+"""
+    StructArrays.staticschema(::Type{<:StaticArray{S, T}}) where {S, T}
+
+The staticschema for a StaticArray is the underlying Tuple used to store the flattened array.
+```julia
+julia> StructArrays.staticschema(SVector{2,Float64})
```

Comment:
```suggestion
julia> StructArrays.staticschema(SVector{2, Float64})
```

Sorry for the nitpicking! It's just to keep the formatting consistent within the source code.

Reply:
No worries! Thanks for going through it so carefully